pyxrf/model/draw_image.py

```diff
@@ -176,15 +175,18 @@ def init_plot_status(self, change):
         self.pixel_or_pos = 0
 
         if 'positions' in self.data_dict:
-            logger.debug('get pos {}'.format(self.data_dict['positions'].keys()))
-            self.x_pos = list(self.data_dict['positions']['x_pos'][0, :])
-            self.y_pos = list(self.data_dict['positions']['y_pos'][:, 0])
-
-            # the only place to flip data is at fileIO
-            # the way to handle position at fileIO is a bit confusing
-            # self.x_pos.reverse()
-            # we use imshow with lower as the origin, so flip y
-            self.y_pos.reverse()
+            try:
+                logger.info('get pos {}'.format(self.data_dict['positions'].keys()))
+                self.x_pos = list(self.data_dict['positions']['x_pos'][0, :])
+                self.y_pos = list(self.data_dict['positions']['y_pos'][:, 0])
+
+                # the only place to flip data is at fileIO
+                # the way to handle position at fileIO is a bit confusing
+                # self.x_pos.reverse()
+                # we use imshow with lower as the origin, so flip y
+                self.y_pos.reverse()
```

Comment:
Good point. There are several reasons why I did it this way. When I transfer database into h5 file, I didn't transfer the position data in a proper way. I plan to correct this during beam shutdown time, not causing more bugs on analysis now. But currently the x/y positions are correct.